app/api/websocket.py

```
@@ -0,0 +1,552 @@
+                )
+
+            await websocket.send_text(json.dumps({
+                "type": "done",
+                "trace_id": trace_id,
+            }, ensure_ascii=False))
+            await websocket.close()
+
+    except WebSocketDisconnect:
+        return
+
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)